qa/L0_client_memory_growth/test.sh

```diff
@@ -116,7 +118,8 @@ for PROTOCOL in http grpc; do
                 RET=1
             fi
 
-            # Log the graph for memory growth and the change between Average and Max memory usage
+            # Log test duration, the graph for memory growth and the change between Average and Max memory usage
+            echo -e "Test Duration: $(($TEST_DURATION / 3600)):$((($TEST_DURATION / 60) % 60)):$(($TEST_DURATION % 60)) (HH::MM::SS)" >> ${GRAPH_LOG}
```

Comment:
Sorry for not being clear enough.

You must use formatted outputs to actually get time reported in HH:MM:SS.
See [here ](https://askubuntu.com/questions/1028924/how-do-i-use-seconds-inside-a-bash-script) for example.

So instead of 0:1:2 the output should be like 00:01:02.

Reply:
I see. Thanks for clarifying this, modifying it now.